modules/jaeger-rn-new-features.adoc

```diff
@@ -2,27 +2,19 @@
 Module included in the following assemblies:
 - rhbjaeger-release-notes.adoc
 ////
+////
+Feature – Describe the new functionality available to the customer.  For enhancements, try to describe as specifically as possible where the customer will see changes.
+Reason – If known, include why has the enhancement been implemented (use case, performance, technology, etc.).   For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.
+Result – If changed, describe the current user experience.
+////
 
 [id="jaeger-rn-new-features_{context}"]
+== New features {ProductName} 2.0.0
 
+* This release of {ProductName} adds support for using an "external" Elasticsearch cluster to store tracing data, that is, an Elasticsearch instance not installed and created by the Elasticsearch Operator.
 
-== New features {ProductName} 1.17.4
-
-This release of {ProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
-
-== New features {ProductName} 1.17.3
-
-This release of {ProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
+* This release adds autoscaling support for the Jaeger Collector and Ingester.
 
-== New features {ProductName} 1.17.2
-
-This release of {ProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
-
-== New features {ProductName} 1.17.1
-////
-Feature – Describe the new functionality available to the customer.  For enhancements, try to describe as specifically as possible where the customer will see changes.
-Reason – If known, include why has the enhancement been implemented (use case, performance, technology, etc.).   For example, showcases integration of X with Y, demonstrates Z API feature, includes latest framework bug fixes.  There may not have been a 'problem' previously, but system behaviour may have changed.
-Result – If changed, describe the current user experience
-////
+* This release enabled support for services or applications running outside of an OpenShift cluster to be able to report tracing data to Jaeger running within the OpenShift cluster.
```

Comment:
As this is via steps described in an article, would be it ok to add a link to the article here? As otherwise it sounds like something has been updated in the product to support this?

Reply:
@JStickler any thoughts on this one?

Reply:
Which article were you thinking of?

Reply:
The one related to https://issues.redhat.com/browse/OSSMDOC-145 (was TRACING-1432).

Reply:
That's the first thing on my To Do list after the GA, so there's nothing I can link to yet.

Reply:
Would it be better to remove/comment out this bullet until the article is ready? At the moment I think it would lead to questions to support asking how to configure this.

Reply:
Yes, I can do that and restore it once we can link to the article.